app/controllers/two_factor_authentication/personal_key_verification_controller.rb

```diff
@@ -25,7 +25,7 @@ def track_analytics(result)
       mfa_created_at = current_user.encrypted_recovery_code_digest_generated_at
       analytics_hash = result.to_h.merge(
         multi_factor_auth_method: 'personal-key',
-        multi_factor_auth_method_created_at: mfa_created_at,
+        multi_factor_auth_method_created_at: mfa_created_at&.strftime('%s%L'),
```

Comment:
does this resolve the issue in a sandbox? I've seen CW convert integers to timestamps, so I'm worried it may not help.

Reply:
oh it did resolve it. I can link you the query in cloudwatch

Reply:
thanks!